devtools/klap-decrypt-requirements.txt

```diff
@@ -0,0 +1,61 @@
+aiohttp==3.9.5
```

Comment:
Instead of pinning the versions, please just add an entry to `README.md` and list direct dependencies there. I think installing `pyshark` is enough?

Reply:
Don't forget to remove this file.